app/models/user.rb

```diff
@@ -120,6 +120,9 @@ def suspend!
       raise 'user_already_suspended'
     end
     update!(suspended_at: Time.zone.now)
```

Comment:
I think we want `user.unique_session_id` which is the current active, singular session. Anything on the `identities` rows won't matter if we null out the `user.unique_session_id`

```suggestion
    OutOfBandSessionAccessor.new(unique_session_id).destroy
    update!(suspended_at: Time.zone.now, unique_session_id: nil)
```